flutter_local_notifications/example/lib/main.dart

```diff
@@ -499,22 +505,51 @@ class _HomePageState extends State<HomePage> {
     var androidPlatformChannelSpecifics = AndroidNotificationDetails(
         'your channel id', 'your channel name', 'your channel description',
         importance: Importance.max, priority: Priority.high, ticker: 'ticker');
-    var platformChannelSpecifics =
-        NotificationDetails(android: androidPlatformChannelSpecifics);
-    await flutterLocalNotificationsPlugin.show(
-        0, 'plain title', 'plain body', platformChannelSpecifics,
+    var platformChannelSpecifics = NotificationDetails(android: androidPlatformChannelSpecifics);
+    await flutterLocalNotificationsPlugin.show(0, 'plain title', 'plain body', platformChannelSpecifics,
         payload: 'item x');
   }
 
+  Future<void> _showFullScreenNotification() async {
+    await showDialog(
+        context: context,
+        child: AlertDialog(
+          title: Text('Turn off your screen'),
+          content: Text('to see the full-screen intent in 3 seconds, press OK and TURN OFF your screen'),
+          actions: <Widget>[
+            FlatButton(
+              child: Text('Cancel'),
+              onPressed: () {
+                Navigator.pop(context);
+              },
+            ),
+            FlatButton(
+              child: Text('OK'),
+              onPressed: () async {
+                Navigator.pop(context);
+
+                await Future.delayed(Duration(seconds: 3));
+
+                var androidPlatformChannelSpecifics = AndroidNotificationDetails(
+                    'your channel id', 'your channel name', 'your channel description',
+                    importance: Importance.max, priority: Priority.max, ticker: 'ticker', fullScreenIntent: true);
+                var platformChannelSpecifics = NotificationDetails(android: androidPlatformChannelSpecifics);
+                await flutterLocalNotificationsPlugin.show(0, 'plain title', 'plain body', platformChannelSpecifics,
+                    payload: 'item x');
+              },
+            )
+          ],
+        ));
+  }
+
   Future<void> _showNotificationWithNoBody() async {
     var androidPlatformChannelSpecifics = AndroidNotificationDetails(
         'your channel id', 'your channel name', 'your channel description',
         importance: Importance.max, priority: Priority.high, ticker: 'ticker');
     var platformChannelSpecifics = NotificationDetails(
       android: androidPlatformChannelSpecifics,
     );
-    await flutterLocalNotificationsPlugin.show(
-        0, 'plain title', null, platformChannelSpecifics,
+    await flutterLocalNotificationsPlugin.show(0, 'plain title', null, platformChannelSpecifics,
```

Comment:
Hmm didn't expect the alignment to change as part of this PR. Can you run the formatter to make sure the code is formatted properly and there aren't any analysis issues?

Reply:
I reverted this change. not sure why IntelliJ did that.
Also fixed a couple analysis issue with docs.